pkg/engine/api/context.go

```diff
@@ -57,12 +58,15 @@ func LoadVariable(logger logr.Logger, jp jmespath.Interface, entry kyvernov1.Con
 	} else {
 		if path != "" {
 			if variable, err := ctx.Query(path); err == nil {
-				output = variable
+				if variable != nil {
+					output = variable
+				}
 			} else if defaultValue == nil {
 				return fmt.Errorf("failed to apply jmespath %s to variable %v", path, err)
 			}
 		}
 	}
+	fmt.Printf("%v\n\n", output)
```

Comment:
I don't think we want to log it out. Not at the default log level at least.

Reply:
@VedRatan - why was this resolved?

Reply:
@realshuting - Actually I had used it for debugging the error, now I've removed it.

Reply:
Still seeing is, did you push the change?

Reply:
Sorry @realshuting forgot to push the changes, sending the changes now.

Reply:
You can check now.